packages/unit-testing/jest-configuration/SuiteCloudJestConfiguration.js

```diff
@@ -13,30 +13,250 @@ const PROJECT_TYPE = {
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/crypto/Decipher.js`,
+	},
+	{
+		module: 'N/crypto/hash',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/crypto/Hash.js`,
+	},
+	{
+		module: 'N/crypto/hmac',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/crypto/Hmac.js`,
+	},
+	{
+		module: 'N/crypto/secretKey',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/crypto/SecretKey.js`,
+	},
+	{
+		module: 'N/currency',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/currency/currency.js`,
+	},
+	{
+		module: 'N/currentRecord',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/currentRecord/currentRecord.js`,
+	},
+	{
+		module: 'N/currentRecord/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/currentRecord/CurrentRecordInstance.js`,
+	},
+	{
+		module: 'N/currentRecord/field',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/record/Field.js`,
+	},
+	{
+		module: 'N/currentRecord/column',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/record/Column.js`,
+	},
+	{
+		module: 'N/currentRecord/sublist',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/record/Sublist.js`,
+	},
+	{
+		module: 'N/dataset',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/dataset/dataset.js`,
+	},
+	{
+		module: 'N/dataset/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/dataset/DatasetInstance.js`,
+	},
+	{
+		module: 'N/dataset/condition',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/dataset/Condition.js`,
+	},
+	{
+		module: 'N/dataset/column',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/dataset/Column.js`,
+	},
+	{
+		module: 'N/dataset/join',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/dataset/Join.js`,
+	},
+	{
+		module: 'N/datasetLink',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/datasetLink/datasetLink.js`,
+	},
+	{
+		module: 'N/datasetLink/instance',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/datasetLink/DatasetLinkInstance.js`,
+	},
+	{
+		module: 'N/email',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/email/email.js`,
+	},
 	{
 		module: 'N/http',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/http/http.js`,
 	},
 	{
-		module: 'N/http/clientresponse',
+		module: 'N/http/clientResponse',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/http/ClientResponse.js`,
 	},
 	{
 		module: 'N/https',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/https.js`,
 	},
 	{
-		module: 'N/https/clientresponse',
+		module: 'N/https/clientResponse',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/ClientResponse.js`,
 	},
 	{
-		module: 'N/https/secretkey',
+		module: 'N/https/secretKey',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/SecretKey.js`,
 	},
 	{
-		module: 'N/https/securestring',
+		module: 'N/https/secureString',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/SecureString.js`,
 	},
+	{
+		module: 'N/https/clientCertificate',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/https/clientCertificate.js`,
```

Comment:
The folder containing clientCertificate.js should be `https/clientCertificate`. There is a duplication of this module that needs to be removed at lines 256-260.
The referenced file needs also to be moved to the proposed folder.

Reply:
Done